src/magento-payments/payoutreports.md

```diff
@@ -0,0 +1,133 @@
+---
+ee_only: false
+title: Payout Reports
+---
+
+Payout reports show comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+You can efficiently cross-reference orders and payments and reconcile accounts---all from one place.
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payout reports in the Admin.
+
+You can download payout reports in a .csv file format for use in existing accounting or order management software.
```

Comment:
What are your thoughts on changing "download" for "export"?

Reply:
The mechanism is known as a download mechanism, and the button graphic is indicative of that and conventionally familiar as such. I'm inclined to leave this.